client/scss/components/_listing.scss

```diff
@@ -27,10 +27,10 @@ ul.listing {
 
   td,
   th {
-    padding: 1.2em 0.3em;
+    padding: calc(1em * var(--w-density-factor)) 0.3em;
```

Comment:
This and below – is a loose interpretation of the designs, as they’re quite out of sync with the build.